### PR TITLE
WIP: test ROR upgrade in omnibus build

### PR DESCRIPTION
**Pull request by RajeshPaul38**
Signed-off-by: Rajesh Paul <rajesh.paul@progress.com>

### Description

[Please describe what this change achieves]

### Issues Resolved

[List any existing issues this PR resolves, or any Discourse or
StackOverflow discussions that are relevant]

### Check List

- [ ] New functionality includes tests
- [ ] All buildkite tests pass
- [ ] Full omnibus build and tests in buildkite pass
- [ ] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/main/CONTRIBUTING.md#developer-certification-of-origin-dco>
- [ ] PR title is a worthy inclusion in the CHANGELOG

### CODE REVIEW

**Comment by RajeshPaul38** [comment]
Build link: https://buildkite.com/chef/chef-supermarket-main-omnibus-adhoc/builds/377
In progress

**Comment by RajeshPaul38** [comment]
Build link: https://buildkite.com/chef/chef-supermarket-main-omnibus-adhoc/builds/380
in progress

**Comment by RajeshPaul38** [comment]
build: https://buildkite.com/chef/chef-supermarket-main-omnibus-adhoc/builds/383
in progress

**Comment by RajeshPaul38** [comment]
New Build: https://buildkite.com/chef/chef-supermarket-main-omnibus-adhoc/builds/384
in progress

**Comment by RajeshPaul38** [comment]
Build: https://buildkite.com/chef/chef-supermarket-main-omnibus-adhoc/builds/385
in progress

**Comment by RajeshPaul38** [comment]
Testing build on ubuntu: https://buildkite.com/chef/chef-supermarket-main-omnibus-adhoc/builds/399
in progress

**Comment by RajeshPaul38** [comment]
Build: https://buildkite.com/chef/chef-supermarket-main-omnibus-adhoc/builds/404

**Comment by RajeshPaul38** [comment]
Build: https://buildkite.com/chef/chef-supermarket-main-omnibus-adhoc/builds/406
in progress

**Comment by RajeshPaul38** [comment]
Build: https://buildkite.com/chef/chef-supermarket-main-omnibus-adhoc/builds/410

**Comment by RajeshPaul38** [comment]
Build: https://buildkite.com/chef/chef-supermarket-main-omnibus-adhoc/builds/418

**Comment by RajeshPaul38** [comment]
Build: https://buildkite.com/chef/chef-supermarket-main-omnibus-adhoc/builds/420

**Comment by antima-gupta** [comment]
@RajeshPaul38 Build is failing, please check this.

**Comment by RajeshPaul38** [comment]
Adhoc Build: https://buildkite.com/chef/chef-supermarket-main-omnibus-adhoc/builds/443

**Comment by RajeshPaul38** [comment]
adhoc build: https://buildkite.com/chef/chef-supermarket-main-omnibus-adhoc/builds/444

**Comment by RajeshPaul38** [comment]
adhoc build: https://buildkite.com/chef/chef-supermarket-main-omnibus-adhoc/builds/448

**Comment by RajeshPaul38** [comment]
adhoc build: https://buildkite.com/chef/chef-supermarket-main-omnibus-adhoc/builds/449

**Comment by RajeshPaul38** [comment]
Both builds in omnibus and buildkite are passing. But would need to use the conventional source for runit instead of github url. Will wait for runit to be pushed to remote for consumption. Can close only after that.

File: omnibus/cookbooks/omnibus-supermarket/Gemfile [hunk 1]
```diff
@@ -8,7 +8,7 @@ gem 'chef', '~> 16.14'
 gem 'chef-bin', '~> 16.14'
 gem 'inspec-core'
 gem 'inspec-core-bin'
-gem 'mini_racer', '~> 0.3.1', platforms: :ruby
+# gem 'mini_racer', '~> 0.4.0', platforms: :ruby
```
Comment (ayush-billore) [inline omnibus/cookbooks/omnibus-supermarket/Gemfile:11]: @RajeshPaul38 In this line we are first updating the version of `mini_racer` and then commenting the line. Can you please provide some context here?

Comment (RajeshPaul38) [reply omnibus/cookbooks/omnibus-supermarket/Gemfile:11]: Yeah mini_racer won't be needed I believe. We have got rid of it from throughout the codebase. Will remove this line.

Comment (RajeshPaul38) [reply omnibus/cookbooks/omnibus-supermarket/Gemfile]: Pushed the changes suggested.

**Review by ayush-billore — APPROVED** [review]

**Comment by RajeshPaul38** [comment]
These changes have already been merged to the target branch through another PR.

**Pull request closed without merging**